docs/src/pages/[platform]/theming/theme-provider/react.mdx

```diff
@@ -0,0 +1,75 @@
+import { Example, ExampleCode } from '@/components/Example';
+
+import { BasicExample } from './examples';
+
+## Usage
+
+Import the `ThemeProvider` and wrap your application with it:
+
+```jsx
+import { ThemeProvider } from '@aws-amplify/ui-react';
+
+export const App = (
+  <ThemeProvider>
+    <YourApplication />
+  </ThemeProvider>
+);
+```
+
+Use the [default theme](./default-theme) to style the components in your app:
```

Comment:
Maybe here we could say something about after wrapping their application in a ThemeProvider they have access to theme values in their components. Because that is what the example is showing (regardless of if they use the default theme or not)